### PR TITLE
Update index.md

### LINKED ISSUE #15797: Kestrel is the default web server included in ASP.NET Core project templates.
Kestrel is a web server - it's not in the templates.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: d8f53b58-a048-69ca-2e89-3625bb4cc839
* Version Independent ID: 32a343a1-ba87-7274-e861-2ac194d5aa3d
* Content: [Web server implementations in ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/servers/?view=aspnetcore-3.0&tabs=windows)
* Content Source: [aspnetcore/fundamentals/servers/index.md](https://github.com/aspnet/AspNetCore.Docs/blob/master/aspnetcore/fundamentals/servers/index.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-fundamentals**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

### PR DESCRIPTION
**Pull request by Rick-Anderson**
Fixes #15797 

cc @guardrex  @serpent5

### CODE REVIEW

**Comment by guardrex** [comment]
There is no direct relationship between `dotnet run` and the server(s) involved.

If anything, I'd rest easier ⚰️ by saying ...

> Kestrel is the default web server specified by the ASP.NET Core project templates.

... the server isn't "included" in the template in any cosmic sense. The template is set up to use it, so it seems more "specified by" than "included in."

**Comment by Rick-Anderson** [comment]
@guardrex  better - but - what about 85% of our customers who create templates and launch from VS, where IIS Express is the default web server?

**Comment by guardrex** [comment]
It's a little further down in the Windows tab ... gets into it.

This has been a pain point from the start with explaining ASP.NET Core. Yes, indeed ... there often hasn't been just one server for most users, but many devs come in looking to hear about the **ONE** server that they'll be using. The word "server" is overloaded. They eventually learn that there are *several servers* around here.

... tack on "host" (two other things now because `IWebHostBuilder` is still a thing with the Generic Host), and "host" is overloaded with "server." Welcome to my world of the *Host and Deploy* node! :smile::cry::smile:

File: aspnetcore/fundamentals/servers/index.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ An ASP.NET Core app runs with an in-process HTTP server implementation. The serv
 
 ## Kestrel
 
-Kestrel is the default web server included in ASP.NET Core project templates.
+Kestrel is the default web server specified by the ASP.NET Core project templates.
```
Comment (Rick-Anderson) [inline aspnetcore/fundamentals/servers/index.md:19]: @guardrex I'm changing this. The templates don't specify Kestrel. Most customers use VS and that launches IIS Express.

**Pull request merged**